src/index.html

```diff
@@ -9,6 +9,62 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="link-img">
+        <img 
+          src="images/logo.png" 
+          class="logo" 
+          alt="0"
+        >
+      </a>
+      <nav class="nav-link">
+        <ul class="list">
+          <li class="li">
+            <a href="#" class="list-active is-active">
```

Comment:
Why do all your links have `active` in the class name?
```suggestion
            <a href="#" class="nav__link is-active">
```